bucket/aspell.json

```
@@ -0,0 +1,62 @@
+{
+    "##": "The dictionary (aspell6-en-2020.12.07-1-x86_64.pkg.tar.zst) has to be updated manually.",
+    "version": "0.60.8-2",
+    "description": "A spellchecker commonly used with emacs.",
+    "homepage": "http://aspell.net/",
+    "license": "LGPL-2.1-only",
+    "architecture": {
+        "64bit": {
+            "url": [
+                "https://mirror.msys2.org/mingw/mingw64/mingw-w64-x86_64-aspell-0.60.8-2-any.pkg.tar.zst",
+                "https://mirror.msys2.org/msys/x86_64/aspell6-en-2020.12.07-1-x86_64.pkg.tar.zst#/dict"
```

```suggestion
                "https://mirror.msys2.org/mingw/mingw64/mingw-w64-x86_64-aspell-en-2020.12.07-1-any.pkg.tar.zst"
```

We should use mingw packages with mingw packages: mixing mingw and msys packages can lead to incompatibilities sometimes.

Also, with this we can use extract_dir as mingw64

---

Thanks for the details!